README.md

```diff
@@ -89,62 +94,94 @@ The RPC methods currently implemented are:
 * `bzz_hive` (stub)
 * `bzz_info` (stub)
 * `debug_traceTransaction` (without support for `tracer` option)
-* `eth_accounts`
-* `eth_blockNumber`
-* `eth_call`
-* `eth_coinbase`
-* `eth_estimateGas`
-* `eth_gasPrice`
-* `eth_getBalance`
-* `eth_getBlockByNumber`
-* `eth_getBlockByHash`
-* `eth_getBlockTransactionCountByHash`
-* `eth_getBlockTransactionCountByNumber`
-* `eth_getCode` (only supports block number “latest”)
-* `eth_getCompilers`
-* `eth_getFilterChanges`
-* `eth_getFilterLogs`
-* `eth_getLogs`
-* `eth_getStorageAt`
-* `eth_getTransactionByHash`
-* `eth_getTransactionByBlockHashAndIndex`
-* `eth_getTransactionByBlockNumberAndIndex`
-* `eth_getTransactionCount`
-* `eth_getTransactionReceipt`
-* `eth_hashrate`
-* `eth_mining`
-* `eth_newBlockFilter`
-* `eth_newFilter` (includes log/event filters)
-* `eth_protocolVersion`
-* `eth_sendTransaction`
-* `eth_sendRawTransaction`
-* `eth_sign`
-* `eth_subscribe` (only for websocket connections. "syncing" subscriptions are not yet supported)
-* `eth_unsubscribe` (only for websocket connections. "syncing" subscriptions are not yet supported)
-* `eth_syncing`
-* `eth_uninstallFilter`
-* `net_listening`
-* `net_peerCount`
-* `net_version`
-* `miner_start`
-* `miner_stop`
-* `personal_listAccounts`
-* `personal_lockAccount`
-* `personal_newAccount`
-* `personal_importRawKey`
-* `personal_unlockAccount`
-* `personal_sendTransaction`
-* `shh_version`
-* `rpc_modules`
-* `web3_clientVersion`
-* `web3_sha3`
-
-There’s also special non-standard methods that aren’t included within the original RPC specification:
-
-* `evm_snapshot` : Snapshot the state of the blockchain at the current block. Takes no parameters. Returns the integer id of the snapshot created. A snapshot can only be used once. After a successful `evm_revert`, the same snapshot id cannot be used again. Consider creating a new snapshot after each `evm_revert` if you need to revert to the same point multiple times.
+* <a href="https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_accounts" target="_blank">eth_accounts</a>
```

Comment:
Any reason not to use markdown links for these instead of HTML anchors?

Reply:
I was hoping to leverage the target property so links would open in a new tab (and not take the user away from our docs).  It didnt seem to work unfortunately, we should definitely switch it back to the markdown markup.